src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -20,12 +20,6 @@ internal class OpsConfigAdapter : IDisposable
         private const string MetadataSchemaApi = "https://ops/metadataschema/";
         private const string MarkdownValidationRulesApi = "https://ops/markdownvalidationrules/";
 
-        private static readonly string s_opsToken = Environment.GetEnvironmentVariable("DOCS_OPS_TOKEN");
```

Comment:
Test should be updated
https://github.com/dotnet/docfx/blob/91b3e55f1a3d28552baed293253542496944c418/docs/specs/config.yml#L527